src/rgw/rgw_opa.cc

```diff
@@ -42,11 +42,14 @@ int rgw_opa_authorize(RGWOp *& op,
   JSONFormatter jf;
   jf.open_object_section("");
   jf.open_object_section("input");
-  jf.dump_string("method", s->info.env->get("REQUEST_METHOD"));
   jf.dump_string("relative_uri", s->relative_uri.c_str());
   jf.dump_string("decoded_uri", s->decoded_uri.c_str());
   jf.dump_string("params", s->info.request_params.c_str());
   jf.dump_string("request_uri_aws4", s->info.request_uri_aws4.c_str());
+  const char *request_method = s->info.env->get("REQUEST_METHOD");
```

Comment:
you should probably move it up to uts original location. so that the order of the output json does not change